### DISCUSSION TITLE
Windows Docker - Using Hardware transcoding possible ?

### DISCUSSION THREAD

**Discussion by juan393939** [photoprism/photoprism#2494] [Q&A]
Hi !

I'm new to Photoprism and non-native english speaker, so my apologies in advance :)

I have a Windows PC with Docker on it, with Photoprism + Traefik, that works perfectly fine so far :)

The main issue i'm having is the delay needed for my PC to transcode videos when i want to see them. (2 minutes of wait before being able to see a 200mb 4k video)

I've read somewhere that i could do it in background with a scheduled task that would run the " docker-compose exec photoprism photoprism convert " command.

But still, i find the transcode process slow. So i've read that sponsors could use the "h264_nvenc" encoder. I wanna give it a try, but i want to know if this is even possible on Windows Docker. I've read that you need to "forward" the GPU device to the PhotoPrism container. However, i've not see any instructions on how to do it on Windows, is this even possible ?

Thanks in advance !

**Comment by centralhardware** [comment]
docker on window is virtual machine whether it's hyper-v or WSL-2 (what really still hyper-v). so you can't passthrough any pci devices without disable it on host. then you can't using Hardware transcoding
